src/main/java/webdriver/screen/yandex/disk/LoginPage.java

```diff
@@ -21,21 +25,34 @@ public LoginPage(WebDriver driver) {
 
     public LoginPage open() {
         browser.getDriver().get(URL);
+        logger.info("Login page opened");
         return this;
     }
 
     public LoginPage typeLogin(String login) {
         browser.typeText(FIELD_FOR_LOGIN_LOCATOR, login);
+        logger.info("Typed username");
```

Comment:
That would be beneficial to log the data that you type logger.info(String.format("Typed: '%s' username", login));